doc/modules/language-guide/pages/basic-concepts.adoc

```diff
@@ -121,7 +121,7 @@ x * y + x;
 
 === Declarations and variable substitution
 
-Declarations can be mutually recursive, but in cases where they are not, they permit a substitution semantics.
+Declarations can be mutually recursive, but in cases where they are not, they permit a substitution of semantics.
```

Comment:
```suggestion
Declarations can be mutually recursive, but in cases where they are not, they permit a substitution semantics  
(that is, replacing equals for equals, as familiar from high-school algebraic simplification).
```

Reply:
@cPayIt, I've made the correction as a suggestion above. You can now commit the suggestion and ask us to review it again.